pythonbits/bb.py

```diff
@@ -930,6 +933,140 @@ def _render_form_description(self):
         return self['description']
 
 
+class BookSubmission(BbSubmission):
+
+    _cat_id = 'book'
+    _form_type = 'E-Books'
+
+    def _desc(self):
+        s = self['summary']
+        return re.sub('<[^<]+?>', '', s['description'])
+
+    @form_field('book_retail', 'checkbox')
+    def _render_retail(self):
+        return bool(
+            input('Is this a retail release? [y/N]  ').lower()
+            == 'y')
+
+    @form_field('book_language')
+    def _render_language(self):
+        return self['summary']['language']
+
+    @form_field('book_publisher')
+    def _render_publisher(self):
+        return self['summary']['publisher']
+
+    @form_field('book_author')
+    def _render_author(self):
+        return self['summary']['authors'][0]['name']
+
+    @form_field('book_format')
+    def _render_format(self):
+        book_format = {
+            'EPUB': 'EPUB',
+            'MOBI': 'MOBI',
+            'PDF': 'PDF',
+            'HTML': 'HTML',
+            'TXT': 'TXT',
+            'DJVU': 'DJVU',
+            'CHM': 'CHM',
+            'CBR': 'CBR',
+            'CBZ': 'CBZ',
+            'CB7': 'CB7',
+            'TXT': 'TXT',
+            'AZW3': 'AZW3',
+            }
```

Comment:
This seems to be just an identity map, why the dict then - in case some future format has a name different from its filename suffix?